starters/graphql-starter/components/drupal/ArticleTeaser.tsx

```diff
@@ -21,20 +20,15 @@ export function NodeArticleTeaser({ node, ...props }: NodeArticleTeaserProps) {
             <span className="font-semibold">{node.author.displayName}</span>
           </span>
         ) : null}
-        <span> - {formatDate(node.created)}</span>
+        {node.created && <span> - {formatDate(node.created)}</span>}
       </div>
       {node.image && (
         <figure className="my-4">
-          <Image
-            src={node.image.url}
-            width={768}
-            height={480}
-            alt={node.title}
-          />
+          <Image src={node.image.url} width={768} height={480} alt="" />
```

Comment:
The node title is _not_ the alt text for its image. But I don't see that data in the GraphQL explorer, so an empty alt text is a more accessible option.